addons/website_slides/views/website_slides_templates_profile.xml

```diff
@@ -39,12 +39,9 @@
                 <div class="card mb-2">
                     <div class="card-body o_wprofile_slides_course_card_body p-0 d-flex"
                         t-attf-onclick="location.href='/slides/#{slug(course.channel_id)}';">
-
-                        <div t-if="course.channel_id.image_1024" class="ps-5 pe-4 rounded-start" t-attf-style="background-image: url(#{website.image_url(course.channel_id, 'image_1024')}); background-size: cover; background-position: center"/>
-                        <div t-else="" class="o_wslides_card_default_background ps-5 pe-4 rounded-start position-relative" style="opacity: 0.8">
-                            <i class="fa fa-graduation-cap fa-fw me-2 mt-3 position-absolute text-white-75" style="right:0; top: 0"/>
-                        </div>
-
+                        <t t-set="background_image"
+                           t-value="website.image_url(course.channel_id, 'image_1024') if course.channel_id.image_1024 else '/website_slides/static/src/img/channel-%s-default.jpg' % ('training' if course.channel_id.channel_type == 'training' else 'documentation')"/>
```

Comment:
I've seen this code a couple times now.
Maybe we could have a "_get_background_image_url" method on the slide.channel model to make it a bit more elegant and avoid code duplication?